test/functional/editor/tabpage_spec.lua

```diff
@@ -53,6 +54,46 @@ describe('tabpage', function()
     neq(999, eval('g:win_closed'))
   end)
 
+  it('no glitch when switching tabpage after setting laststatus=3 #19591', function()
```

Comment:
"no glitch", "correct", "works" all are redundant in tests.

```suggestion
  it('switching tabpage after setting laststatus=3 #19591', function()
```